### PR TITLE
Update alerts.md

### PR DESCRIPTION
**Pull request by tiffany-mixpanel**
updated docs to include anomaly detection, this is original notion file:

https://www.notion.so/mxpnl/Alerts-help-doc-updated-w-Anomaly-Detection-ed2840af15d94c6f80145afacea8fa7e?pvs=4

### CODE REVIEW

File: pages/docs/analysis/advanced/alerts.md [hunk 1]
```diff
@@ -1,74 +1,119 @@
...
 
-Custom alerts in Insights alert you when a specified event is performed above or below a specified number.
+<img src="https://github.com/mixpanel/docs/assets/130006730/91a2bb99-3a79-43d4-9d97-3b397aab0305" width="400" height="whatever">
 
-Enter a name for your alert and select an event to track. Select **above** or **below** and the value you want as the threshold.
 
-- **Above:** The alert will check yesterday’s full day of data compared to today’s data, checking every hour. For example, at 5 pm the alert will check 12 am - 5 pm, and if the current value is above the previous value, an alert will be sent and the data won’t be checked again for another 24 hours.
-- **Below:** The alert will compare the previous day’s value with the day before. If the new value is below the previous value, you will be sent an alert and the data won't be checked again for another 24 hours. If the value is above the previous value, you won't receive an alert.
+## **View & Manage Custom Alerts**
```
Comment (mavlee) [inline pages/docs/analysis/advanced/alerts.md]: don't need these **, the heading already does this

File: pages/docs/analysis/advanced/alerts.md [hunk 2]
```diff
@@ -1,74 +1,119 @@
...
 
-Enter the recipient emails that you would like to receive this alert. You can enter any email here, not just internal users. Select any Slack channels you want to send this alert to. Private Slack Channels are not available at this time.
+1. Click on **the gear icon** in the top right of Mixpanel and under "Organization Settings" **select "Personal Settings".
+2. Within your personal settings, select the **Alerts** tab to view your custom alerts and automatic insights.
+    
+    **to delete:** If you hover over an alert you have created, you can see a **trash icon** which you can click to delete the alert. 
+    **to modify:** Click on the title of the alert to edit details.
 
-![/14210649496084](/14210649496084.png)
+![image](https://github.com/mixpanel/docs/assets/130006730/be35a381-b090-4fd4-86e0-7281dd291d64)
```
Comment (mavlee) [inline pages/docs/analysis/advanced/alerts.md:40]: this image should be uploaded to the repo

File: pages/docs/analysis/advanced/alerts.md [hunk 3]
```diff
@@ -1,74 +1,119 @@
...
 
-Click **Create alert** to save the custom alert.
+You can view the details of another user’s alert, but you are not able to edit or delete it unless you're a project owner.
 
-**IMPORTANT:** You may need to add Mixpanel to Slack, if you have not done this before. Click "Add Mixpanel to Slack" and follow the prompts on the following screens. You will be brought back to this page when you have added Mixpanel to Slack.
+***IMPORTANT: If you need to modify alerts you didn't create, contact the project owner.***
 
-![/14251383457684](/14251383457684.png)
+# Alert Criteria Types
```
Comment (mavlee) [inline pages/docs/analysis/advanced/alerts.md]: this should be an h2

File: pages/docs/analysis/advanced/alerts.md [hunk 4]
```diff
@@ -1,74 +1,119 @@
...
 
-### Line Charts
+## Anomaly Detection
```
Comment (mavlee) [inline pages/docs/analysis/advanced/alerts.md]: which makes this h3

File: pages/docs/analysis/advanced/alerts.md (hunk 2 above)
Comment (ishamehramixpanel) [reply pages/docs/analysis/advanced/alerts.md:40]: ok to paste in

**Pull request merged**